### PR TITLE
#71 updated current and prev tech leads; replaced raw html with the react components using .mdx

### LINKED ISSUE #71: React component for profile pictures
## Overview 

Profile pictures on https://github.com/oss-slu/oss-slu.github.io/blob/main/docs/about/about.md should be presented with a React component rather than raw HTML. The component should link to their GitHub profile and be able to pull either a local file or use a GitHub profile picture. The page should be updated to use the component.

## AC
* React component that creates a profile picture identical to current format
  * links to the user's GitHub profile
  * can use a local file for the image
  * fallback to use a GitHub profile picture if no local file
* "About Us" page is updated to use the component

### PR DESCRIPTION
**Pull request by Anjali0407-git**
Fixes #71

After making changes:

image

### CODE REVIEW

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
I have one small change, and I want to check my assumptions. 

First, I think it makes sense to update the profile picture for Dr. Holdener and myself using the same `Profile` component, but I think it might make more sense to directly replace the `<div>` tag with it. What do you think?

That leads to my next question, a check on my assumptions. I was originally thinking of directly replacing `<div>`-based profiles with the `Profile` component directly in the .mdx file. My thinking was it is obvious what is doing what, and making updates is just editing a slightly fancy markdown file. Your approach separates the MDX  and component as a presentation of a separate set of data. That's still pretty clean, and updates are just as quick and easy. It forces a preservation of the overall layout and structure of the page, too. My only concern is that it might add to the complexity, and therefore add to potential confusion when we go back to update it. What do you think?

As I'm looking at the overall page layout there are some other elements that probably need restructuring, too. I think it's worth getting this done before moving on to those, though.

**Comment by Anjali0407-git** [comment]
I think replacing the `<div>` tags with `Profile `component won't actually reduce the number of lines of code we use. Also, since we'll need to keep adding new student profiles every semester, the way we're doing it now might be easier. As the Current Tech leads, Previous Tech leads and Developers does not have the individual anchor tags to worry about. 
However, for important roles like the Leadership Team or SLU research Tech Leads, using the Profile component seem like the best option because it has your names as the title which is an anchor tag. Not sure if we can replicate the same behaviour using React

File: src/data/people/currentGradTechLeads.json
Comment (kungfuchicken) [inline src/data/people/currentGradTechLeads.json:1]: Add everyone here, and I think it's ready to ship.

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
If you need help generating the full list of current leads, i can help, but if I push up the change GH won't let me give a clean approval.

File: src/data/people/currentGradTechLeads.json
Comment (Anjali0407-git) [reply src/data/people/currentGradTechLeads.json:1]: Yes, I just addressed it by adding other 17 current tech leads there. And using the github-image as the profile image incase when image is not available in this json file

**Review by kungfuchicken — APPROVED** [review]
onward and upward! lgtm;

**Pull request merged**